storybookjs/storybook
# Core: Sync args state to URL

Issue: #12291

## What I did

This synchronizes args state with the URL.

## How to test

- Is this testable with Jest or Chromatic screenshots? No, but a Cypress test could.
- Does this need a new example in the kitchen sink apps? No
- Does this need an update to the documentation? Maybe

If your answer is yes to any of these, please make sure to include it in your PR.
